### DISCUSSION TITLE
Author links in New Tab

### DISCUSSION THREAD

**Discussion by sergiovaneg** [mmistakes/minimal-mistakes#4838] [Q&A]
First of all, I want to say working with this theme and its documentation makes me realise why it's the first one in its category.

Is there an idiomatic way to open the social media links in a new tab instead of navigating away from my webpage? I want to avoid overwriting the default HTML layout files since it would become inconvenient to sync any updates to the theme with my site.

**Comment by iBug** [comment]
Depending where the "social media links" are located, you need to copy the appropriate template file from the theme, and add `target="_blank"` on those links you want to open in a new tab. If you don't want to override theme files, consider writing some JavaScript as an alternative.

**Reply by sergiovaneg** [reply]
Thanks! Is there a way to embed classic html into the links list inside the author block? If not, I'll look into the JS solution since I prefer adding some code rather than de-syncing the theme.

**Reply by iBug** [reply]
Adding HTML should be as straightforward as it needs to be.
The navigation `title`-s are rendered verbatim so any HTML you put in `navigation.yml` should render.
([Example](https://github.com/iBug/iBug-source/blob/3bb753cecaf79dfbfade88b0edddcc2ac306cc35/_data/navigation.yml#L18) on my website)
